drpayyne/magento-cloud-docker
# Added Blackfire support

### Description

Added Blackfire support for magento cloud docker development

### Fixed Issue

-  magento/magento-cloud-docker#131: Add Blackfire.io to Cloud Docker

### Manual testing scenarios
1. Add Blackfire configuration to `.magento.app.yaml` under `runtime.extensions` as described below.  
    ```
    runtime:
      extensions:
        - random-extension-here
        -
          name: blackfire
          configuration:
            server_id: SERVER_ID
            server_token: SERVER_TOKEN
            client_id: CLIENT_ID
            client_token: CLIENT_TOKEN
    ```
2. Generate customized (Blackfire supported) PHP images and Docker configuration as described in [docker configuration generator](https://devdocs.magento.com/cloud/reference/ece-tools-reference.html#docker-configuration-generator) and [docker development](https://devdocs.magento.com/cloud/docker/docker-development.html).
3. Follow the rest of the guide linked [above](https://devdocs.magento.com/cloud/docker/docker-development.html) by bringing up the containers and deploying Magento.
4. Ensure necessary containers are up and running.
5. Install a profiling client as described in [Blackfire documentation](https://blackfire.io/docs/up-and-running/installation#install-a-profiling-client).
6. Profile the Magento website.

### Contribution checklist
